lib/cli/commands/build.js

```diff
@@ -52,28 +52,38 @@ optimized output compatible with Node 6.
     });
   }
 
-  build(dirs = this.dirs) {
-    if (!dirs) {
-      dirs = this.dirs = [ 'app', 'config' ];
+  build(globs = this.globs) {
+    if (!globs) {
+      globs = this.globs = [
+        'app/**/*',
+        'config/**/*',
+        'blueprints/*/index.js',
+        { glob: 'blueprints/*/files/**/*', transform: false }
```

Comment:
@davewasmer I did this in the mean time. We can explore broccolifying it in another PR.